src/event.js

```diff
@@ -474,10 +474,12 @@ jQuery.event = {
 		},
 		click: {
 
-			// For checkbox, fire native event so checked state will be right
+			// For checkable types, fire native event so checked state will be right
 			trigger: function() {
-				if ( this.type === "checkbox" && this.click && jQuery.nodeName( this, "input" ) ) {
+				if ( rcheckableType.test( this.type ) &&
+					this.click && jQuery.nodeName( this, "input" ) ) {
 					this.click();
+
```

Comment:
This blank is one line too low (it should follow the closing brace of the multi-line `if` condition, to separate it from its body).